__tests__/app.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest')
+const app = require('../app')
+const connection = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(()=> {
+    
+    return seed(testData)
+})
+afterAll(()=> {
+    connection.end()
+})
+
+
+describe('app', () => {
+    describe('GET /api', () => {
+        test('200: GET responds with a message of root ok', () => {
+            return request(app)
+            .get('/')
+            .expect(200)
+            .then((body) => {
```

Comment:
Had another look at this and you can pop body in {} i.e. {body}, then just do body.msg - should work.